src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/DocumentChangedSynchronizationService.cs

```diff
@@ -53,9 +53,9 @@ public void RazorDocumentChanged(RazorFileChangeEventArgs args)
             var projectFilePath = args.UnevaluatedProjectInstance.ProjectFileLocation.File;
             var documentFilePath = args.FilePath;
 
-            Task.Factory.StartNew(
+            _ = Task.Factory.StartNew(
```

Comment:
I could take or leave the "no implicit discard" analyzer (though for tasks specifically it makes sense).

Reply:
Well in this case it's this instead of "await xyz"